hack/golang-version.sh

```diff
@@ -15,7 +15,7 @@
 
 SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../hack && pwd )"
 
-DOCKERFILE_ROOT=${SCRIPTS_DIR}/../deployments/devel
+DOCKERFILE_ROOT="${SCRIPTS_DIR}/.."
```

Comment:
Question: Given that we use `FROM golang:1.23-bookworm AS builder`, does this properly extract the version?